dex-core/src/ErgoDex/Class.hs

```diff
@@ -8,3 +8,6 @@ class FromLedger a where
 
 class ToLedger a where
   toLedger :: a -> TxOutCandidate
+
+class FromExplorer a b where
+  parseFromExplorer :: a -> Maybe b
```

Comment:
Core module should not have any dependencies on explorer. Makes sense to put a TC like this into explorer module.

Reply:
moved into quickblue module